android/viewsets/lexicon/src/main/java/com/liferay/mobile/screens/viewsets/lexicon/util/FormViewUtil.java

```diff
@@ -27,25 +28,20 @@
  */
 public class FormViewUtil {
 
-    public static void setupTextFieldLayout(Context context, boolean valid, TextView labelTextView,
-        EditText textEditText) {
+    public static void setupBackground(Drawable drawable, View view) {
```

Comment:
Why just not call `view.setBackgroundDrawable` instead of wrapping the call in a function?